plugins/GSdx/GSTextureCache.cpp

```diff
@@ -480,6 +480,10 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 			GSVector2i custom_resolution = m_renderer->GetCustomResolution();
 			scale_factor.x = static_cast<float>(custom_resolution.x) / width;
 			scale_factor.y = static_cast<float>(custom_resolution.y) / height;
+
+			//Don't bother setting a scale factor when people request same resolution as native resolution.
+			if(custom_resolution == GSVector2i(width, height))
+				scale_factor = GSVector2(0,0);
```

Comment:
After looking more into it, I think ``SetScale()`` is quite free. so not worth adding a check to prevent a call. I removed the third commit.